x-pack/platform/plugins/shared/osquery/common/api/packs/find_packs.schema.yaml

```
@@ -5,14 +5,57 @@ info:
 paths: { }
 components:
   schemas:
-    FindPacksRequestQuery:
+    FindPacksResponse:
       type: object
-      properties:
-        page:
-          $ref: '../model/schema/common_attributes.schema.yaml#/components/schemas/PageOrUndefined'
-        pageSize:
-          $ref: '../model/schema/common_attributes.schema.yaml#/components/schemas/PageSizeOrUndefined'
-        sort:
-          $ref: '../model/schema/common_attributes.schema.yaml#/components/schemas/SortOrUndefined'
-        sortOrder:
-          $ref: '../model/schema/common_attributes.schema.yaml#/components/schemas/SortOrderOrUndefined'
+      properties: { }
+      example:
+        page: 1
+        pageSize: 10
+        total: 1
+        data:
+          - type: 'osquery-pack'
+            id: '42ba9c50-0cc5-11ed-aa1d-2b27890bc90d'
+            namespaces:
+              - 'default'
+            attributes:
+              name: 'My Pack'
+              queries:
+                - query: 'select * from uptime;'
+                  interval: '3600'
+                  id: 'uptime'
+                  ecs_mapping:
+                    - host.uptime:
+                        field: 'total_seconds'
+              enabled: true
+              created_at: '2023-10-31T00:00:00Z'
+              updated_at: '2023-10-31T00:00:00Z'
+              created_by: 'elastic'
+              updated_by: 'elastic'
+              description: 'My pack description'
+        policy_ids: []
+    FindPackResponse:
+      type: object
+      properties: { }
+      example:
+        data:
+          id: "bbe5b070-0c51-11ed-b0f8-ad31b008e832"
```

```suggestion
          id: "3c42c847-eb30-4452-80e0-728584042334"
```

to match the value from the request